store/tikv/gcworker/gc_worker.go

```diff
@@ -104,7 +104,8 @@ const (
 	gcDefaultLifeTime        = time.Minute * 10
 	gcSafePointKey           = "tikv_gc_safe_point"
 	gcSafePointCacheInterval = tikv.GcSafePointCacheInterval
-	gcScanLockLimit          = 1000
+	// we don't want gc to sweep out the cached info belong to other processes, like coprocessor
```

Comment:
Why gc will sweep out the cached info?

Reply:
gc may need much transaction info even if in one scan-lock request, like 1000 or more, so other transaction info will be eliminated.

Reply:
Capital the first letter and add a dot at the end of comment sentence.

Reply:
ok